packages/docusaurus-theme-openapi-docs/src/theme/ApiDemoPanel/Curl/_Curl.scss

```diff
@@ -0,0 +1,4 @@
+.openapi-demo__code-block code {
```

Comment:
Wondering if we can just remove these styles from the container, but we can clean that up later